src/main/java/com/epam/controller/ToursController.java

```diff
@@ -32,6 +34,26 @@ public ModelAndView getToursList() {
         return toursModel;
     }
 
+    @GetMapping("/listoftours/{pageNum}")
+    public ModelAndView getToursListByPage(@PathVariable Integer pageNum, ModelMap modelMap) {
+        ModelAndView toursModel = new ModelAndView();
```

Comment:
it seems like business logic, please replace it to the service layer

Reply:
Don't understand what I should do here

Reply:
Moved some logic to service layer